client-js/README.md

```diff
@@ -0,0 +1,202 @@
+# CEP-78 JavaScript Client Tutorial
+
+This tutorial outlines the usage of the JavaScript client available for the CEP-78 Enhanced NFT Standard.
+
+Further information on the CEP-78 Enhanced NFT Standard can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+## Installing Dependencies
+
+Dependencies can be installed using the following command in the `client-js` directory:
+
+```js
+npm i
+```
+
+## Installing a CEP-78 Contract using the JavaScript Client
+
+The `install` method crafts a [Deploy](https://docs.casperlabs.io/design/casper-design/#execution-semantics-deploys) using `InstallArgs` and sends that Deploy to the specified Casper network.
```

Comment:
```suggestion
The `install` method crafts a [Deploy](https://docs.casperlabs.io/design/casper-design/#execution-semantics-deploys) using `InstallArgs`.
As every deploy created by the SDK you can send it using `.send(rpcUrl)` method providing rpc url that you want to use. It will return deployHash. 
```